helpers/filterSeats.js

```diff
@@ -86,8 +111,56 @@ module.exports = function(dust) {
       });
     }
 
+    function reorderRates(rates, replyIndex) {
+      var bandTypes = {
+        'gold': [],
+        'silver': [],
+        'bronze': []
+      };
```

Comment:
@owennicol 
Just added the change for making that object dynamic in this other PR:
https://github.com/holidayextras/the-wall-templater-dustjs/pull/19/files